server/package.json

```diff
@@ -35,11 +35,6 @@
     "prettier": "1.17.1",
     "request": "2.88.0",
     "rimraf": "2.6.3",
-    "tslint": "5.16.0",
-    "tslint-config-prettier": "1.18.0",
-    "tslint-plugin-prettier": "2.0.1",
-    "tslint-react": "4.0.0",
-    "tslint-react-hooks": "2.1.0",
```

Comment:
Are the hook rules preserved with ESlint? They are incredibly useful when working with hooks :)